text-messaging/send_sms.js

```diff
@@ -0,0 +1,127 @@
+const CronJob = require('cron').CronJob;
+const CronTime = require('cron').CronTime;
+const Messenger = require('../modules/Messenger.js');
+const fetch = require("node-fetch");
+const loadTextReminders = require('./getTextReminders.js').getTextReminders;
+
+var cronJobs = {};
+
+module.exports = (  ) => {
+
+  console.log("In send_sms.js ");
+
+  //initial start of jobs 
+  main();
+
+  //at midnight, recheck the jobs 
```

Comment:
@donovanope Did you end up getting this part working?

Reply:
It works but when it runs on Heroku for like 30 minutes with nothing happening all processes get shut down. I'm looking up what to do to fix it. But other than that, it should work fine.